src/Pages/PeoplePage.tsx

```diff
@@ -0,0 +1,44 @@
+import { useEffect, useState } from 'react';
+import { useParams } from 'react-router-dom';
+import { Person } from '../types';
+import { getPeople } from '../api';
+import { Loader } from '../components/Loader';
+import { PeopleTable } from '../components/PeopleTable';
+
+export const PeoplePage = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [isLoading, setIsLoading] = useState(false);
+  const [errorMessage, setErrorMessage] = useState(false);
```

Comment:
The `errorMessage` state is initialized as a boolean. If `PeopleTable` expects a string or more detailed error message, consider changing this state to store a string message instead of a boolean.